src/main/java/log_in/drivers/LogInDatabaseAccess.java

```diff
@@ -64,17 +64,24 @@ public boolean checkPasswordMatches(String email, String pass){
     }
 
     @Override
-    public ArrayList<String> getUser(boolean success, String email, String pass){
-        ArrayList<String> userInfo = new ArrayList<>();
+    public String[] getUser(boolean success, String email, String pass){
```

Comment:
the boolean success is not needed as the logininteractor already checks this, it can be removed.